PAM_app/app/src/main/java/com/example/pam_app/LoginActivity.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.pam_app
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.content.Intent
+import android.widget.Button
+import android.widget.EditText
+
+
+class LoginActivity : AppCompatActivity() {
+
+    lateinit var sendButton: Button
+    lateinit var nameEditText: EditText
+    lateinit var phoneEditText: EditText
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_login)
+
+        nameEditText = findViewById(R.id.editTextTextPersonName)
+        phoneEditText = findViewById(R.id.editTextPhone)
+        sendButton = findViewById(R.id.sendButton)
```

Comment:
Change init of activity view to android view binding https://developer.android.com/topic/libraries/view-binding

Also change access of view elements to view binding access